tests/unit/test_main.py

```diff
@@ -187,6 +187,33 @@ def test_install_flows_with_delete_strict(self, *args):
         mock_send_napp_event.assert_called_with(self.switch_01, flow,
                                                 'delete_strict')
 
+    @patch('napps.kytos.flow_manager.main.Main._install_flows')
+    def test_event_add_flow(self, mock_install_flows):
+        """Test method for installing flows on the switches through events."""
+        dpid = "00:00:00:00:00:00:00:01"
+        switch = get_switch_mock(dpid)
+        self.napp.controller.switches = {dpid: switch}
+        mock_flow_dict = MagicMock()
+        event = get_kytos_event_mock(name='kytos.flow_manager.flows.install',
+                                     content={'dpid': dpid,
+                                              'flow_dict': mock_flow_dict})
+        self.napp.event_add_flow(event)
+        mock_install_flows.assert_called_with('add', mock_flow_dict, [switch])
+
+    @patch('napps.kytos.flow_manager.main.Main._install_flows')
+    def test_event_add_flow_delete(self, mock_install_flows):
+        """Test method for removing flows on the switches through events."""
+        dpid = "00:00:00:00:00:00:00:01"
+        switch = get_switch_mock(dpid)
+        self.napp.controller.switches = {dpid: switch}
+        mock_flow_dict = MagicMock()
+        event = get_kytos_event_mock(name='kytos.flow_manager.flows.delete',
+                                     content={'dpid': dpid,
+                                              'flow_dict': mock_flow_dict})
+        self.napp.event_add_flow(event)
+        mock_install_flows.assert_called_with('delete', mock_flow_dict,
+                                              [switch])
```

Comment:
Please modify the test names and method calls to reflect the method rename suggested above.

Reply:
Resolved in https://github.com/kytos/flow_manager/pull/129/commits/6876bbce163dce2ebdf3ba9f53c06efaf3b73e33